test/taoensso/tempel_tests.clj

```diff
@@ -84,43 +84,107 @@
 (comment (with-rand-data 32 64 (fn [ba-cnt ?ba-aad] [(is true) (is false) (is true)])))
 
 (deftest _symmetric-cipher-kit
-  (let [sck    (impl/as-symmetric-cipher-kit :aes-gcm-128-v1)
-        ba-key (as-ba 32 "pwd")]
+  (let [aes-gcm-sck (impl/as-symmetric-cipher-kit :aes-gcm-128-v1)
+        ba-aes-key  (as-ba 32 "pwd")
+        chacha-poly-sck    (impl/as-symmetric-cipher-kit :chacha20-poly1305-v1)
+        ba-chacha-poly-key (as-ba 32 "pwd")
+        ;; Chacha2020-Poly1305 JDK implementation doesn't allow initializing the
```

Comment:
Thanks for the clear info on this @iarenaza 👍 Will take a closer look and consider alternatives when I'm ready to merge.